site/_scss/site/objects/_post.scss

```diff
@@ -131,4 +131,16 @@
   img {
     max-width: 100%;
   }
-}
\ No newline at end of file
+}
+
+table {
```

Comment:
This is only for blog posts, not documentation, correct? I don't think we've used tables in any blog posts yet, can you describe the change here?

Reply:
This was to fix tables not having boarders on the Contour envoy resource page, I included as part of this PR to keep layouts the same between OSS projects. 

I have noticed a side affect of this section of CSS on the Contour page, I'll resolve that and update this PR as well so it doesn't come into play. 

Comparison on the Contour page, left is without the snippet and right is the live site after the snippet was applied.

![image](https://user-images.githubusercontent.com/25494777/70087562-aa6e8100-1668-11ea-8a9a-6dc8096fab85.png)

Reply:
That looks a lot better!